astropy/coordinates/baseframe.py

```diff
@@ -637,14 +641,16 @@ def _infer_repr_info(cls, repr_info):
         return repr_info
 
     @classmethod
-    def _create_readonly_property(cls, attr_name, value):
+    def _create_readonly_property(cls, attr_name, value, docstring=None):
         private_attr = '_' + attr_name
 
         def getter(self):
             return getattr(self, private_attr)
 
         setattr(cls, private_attr, value)
         setattr(cls, attr_name, property(getter))
+        if docstring:
```

Comment:
This check will evaluate false for ``docstring=''``. Perhaps the more explicit ``if docstring is not None``

Reply:
Now I just pass it straight on to `property(doc=...)` without any check